src/Modals/ModalMessage.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { fromTheme, VARIANT } from '../Theme';
+
+export interface ModalMessageProps {
+  /** The id of the modal message */
+  id?: string;
+  /** Text to be displayed */
+  children: string;
+  /** Allows you to pass in a variant property from the VARIANT enum */
+  variant: VARIANT;
```

Comment:
Should this prop default to `VARIANT.BASE`?

Reply:
@rmainseas Good idea! I added this in 74d7a33.